internal/lsp/server.go

```diff
@@ -64,6 +64,10 @@ func (s serverState) String() string {
 	return fmt.Sprintf("(unknown state: %d)", int(s))
 }
 
+type ServerConfig struct {
+	UseOwnFormatter bool
```

Comment:
I find the adjective too specific. How about:
```suggestion
	UseFormatter bool
```
or 
```suggestion
	FormatTTCN3 bool
```